README.md

```diff
@@ -54,27 +54,35 @@ Some systems provide pre-built packages:
   $ brew install s3fs
   ```
 
-Compilation
------------
+Compilation and installation from sources
+-----------------------------------------
 
-* On Linux, ensure you have all the dependencies:
+These are generic instructions to compile from the master branch, and should work on almost any GNU/Linux, Mac OS, BSD or similar.
 
-On Ubuntu 14.04:
+If you want specific instructions for some distributions, check the [wiki](https://github.com/s3fs-fuse/s3fs-fuse/wiki/Installation-Notes).
 
-```
-sudo apt-get install automake autotools-dev fuse g++ git libcurl4-openssl-dev libfuse-dev libssl-dev libxml2-dev make pkg-config
-```
+Keep in mind using the pre-built packages when available.
 
-Then compile from master via the following commands:
+1. Ensure your system satisfies build and runtime dependencies for:
 
-```
-git clone https://github.com/s3fs-fuse/s3fs-fuse.git
-cd s3fs-fuse
-./autogen.sh
-./configure
-make
-sudo make install
-```
+   * fuse >= 1.84
```

Comment:
Should this be 2.8.4?

Reply:
Yes, it was a typo: I copied and pasted s3fs-fuse version instead of fuse version.

Fixed now.